### PR TITLE
Add a id_type in from_horizons method and rewrite notebook

### PR DESCRIPTION
**Pull request by shreyasbapat**
Happy New Year! 
2019's first PR (xD)

I noticed that for major bodies, We had to pass `id_type` in `astroquery.jplhorizons.Horizons` . So I added that here.

### CODE REVIEW

File: src/poliastro/twobody/orbit.py [hunk 1]
```diff
@@ -238,14 +238,18 @@ def from_body_ephem(cls, body, epoch=None):
         return ss
 
     @classmethod
-    def from_horizons(cls, name, epoch=None, plane=Planes.EARTH_EQUATOR):
+    def from_horizons(
+        cls, name, epoch=None, plane=Planes.EARTH_EQUATOR, id_type="smallbody"
```
Comment (Juanlu001) [inline src/poliastro/twobody/orbit.py]: I don't like to expose this to the user, but the notebook doesn't work otherwise. A necessary evil, I guess!

**Comment by Juanlu001** [comment]
Oops, I clicked the wrong button. I was about to merge, so I will force push to your branch myself.

File: src/poliastro/twobody/orbit.py (hunk 1 above)
Comment (shreyasbapat) [reply src/poliastro/twobody/orbit.py]: Not just notebook, user won't be able to get the orbit of `major body` if the parameter is not set.

**Comment by shreyasbapat** [comment]
Cool! No issues :grin:

**Comment by Juanlu001** [comment]
Merging! Thanks @shreyasbapat :grin:

**Pull request merged**